CodeAssignment_MVVM_Combine/Sources/Views/ReadySearchView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  EmptyResultView.swift
+//  CodeAssignment_MVVM_Combine
+//
+//  Created by gyusoku.i on 2024/05/27.
+//
+
+import UIKit
+import SnapKit
+
+final class ReadySearchView: UIView {
+    private let backgroundView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .secondarySystemGroupedBackground
+        return view
+    }()
+    
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.image = UIImage(systemName: "doc.text.magnifyingglass")?.withTintColor(.systemGray2, renderingMode: .alwaysOriginal)
+        return imageView
+    }()
+    
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 20, weight: .semibold)
+        label.text = "GitHubのリポジトリが検索できる"
```

Comment:
日本語の問題だけど、「できる」だとちょっと突き放しているように聞こえちゃうので「できます」とかの方がよさそう
```suggestion
        label.text = "GitHubのリポジトリが検索できます"
```

Reply:
[da98047](https://github.com/KyusokLee/CodeAssignment_MVVM_Combine/pull/8/commits/da980474a1bd05ad3e6a526b9e5ad00406551cec)にてコード修正しました。ご確認お願いします！